server/lib/src/server/mod.rs

```diff
@@ -681,6 +683,7 @@ pub trait QueryServerTransaction<'a> {
                         }),
                     SyntaxType::AuditLogString => Ok(PartialValue::new_utf8s(value)),
                     SyntaxType::EcKeyPrivate => Ok(PartialValue::SecretValue),
+                    SyntaxType::Image => Ok(PartialValue::new_image(value)),
```

Comment:
Should probably actually PartialValue to utf8 to match on the tags. PartialValue doesn't have ta equate to the Value itself, since it's just a selector. Look at schema.rs validate for PartialValue. You'll see that Oauth2 accepts Refer as PartialValue for example.

Reply:
I'm not sure what you mean by "match on the tags" here :)